files/ko/web/api/web_audio_api/advanced_techniques/index.html

```diff
@@ -248,66 +248,66 @@ <h3 id="The_final_playPulse_function">The final playPulse() function</h3>
 
 <pre class="brush: js">const bufferSize = audioCtx.sampleRate * noiseLength;
 const buffer = audioCtx.createBuffer(1, bufferSize, audioCtx.sampleRate);</pre>
 
-<p>Now we can fill it with random numbers between -1 and 1:</p>
+<p>이제 우리는 이것을 -1과 1 사이의 랜덤 값으로 채울 수 있습니다.</p>
 
-<pre class="brush: js">let data = buffer.getChannelData(0); // get data
+<pre class="brush: js">let data = buffer.getChannelData(0); // 데이터 얻기
 
-// fill the buffer with noise
+// 버퍼를 노이즈로 채우기
 for (let i = 0; i &lt; bufferSize; i++) {
     data[i] = Math.random() * 2 - 1;
 }</pre>
 
 <div class="note">
-<p><strong>Note</strong>: Why -1 to 1? When outputting sound to a file or speakers we need to have a number to represent 0db full scale — the numerical limit of the fixed point media or DAC. In floating point audio, 1 is a convenient number to map to "full scale" for mathematical operations on signals, so oscillators, noise generators and other sound sources typically output bipolar signals in the range -1 to 1. A browser will clamp values outside this range.</p>
+<p><strong>참고</strong>: 왜 -1에서 1인가요? 파일이나 스피커에 사운드를 출력할 때 우리는 0db 풀 스케일을 표현하기 위해 하나의 숫자를 가질 필요가 있습니다 — 고정점 미디어나 DAC의 숫자로 나타낸 한도입니다. 부동점 오디오에서, 1은 신호에 대한 수학적 연산을 위한 "풀 스케일"을 매핑하는 편리한 숫자입니다. 그래서 오실레이터, 노이즈 생성기, 그리고 다른 음원은 보통 -1에서 1 범위에서 쌍극(bipolar) 신호를 출력합니다. 브라우저는 이 범위 바깥의 값들을 고정시킬 것입니다.</p>
 </div>
 
-<h3 id="Creating_a_buffer_source">Creating a buffer source</h3>
+<h3 id="Creating_a_buffer_source">버퍼 소스 생성하기</h3>
 
-<p>Now we have the audio buffer and have filled it with data, we need a node to add to our graph that can use the buffer as a source. We'll create a {{domxref("AudioBufferSourceNode")}} for this, and pass in the data we've created:</p>
+<p>이제 우리는 오디오 버퍼를 가지고 있고 이것을 데이터로 채웠으니, 우리는 소스로서 버퍼를 사용할 수 있는 그래프에 추가할 노드가 필요합니다. 우리는 이것을 위해 {{domxref("AudioBufferSourceNode")}}를 생성하고, 우리가 생성한 데이터를 전달할 것입니다:</p>
 
 <pre class="brush: js">let noise = audioCtx.createBufferSource();
 noise.buffer = buffer;</pre>
 
-<p>If we connect this through our audio graph and play it —</p>
+<p>만약 우리가 이것을 오디오 그래프를 통해 연결시키고 재생한다면 —</p>
 
 <pre class="brush: js">noise.connect(audioCtx.destination);
 noise.start();</pre>
 
-<p>you'll notice that it's pretty hissy or tinny. We've created white noise, that's how it should be. Our values are running from -1 to 1, which means we have peaks of all frequencies, which in turn is actually quite dramatic and piercing. We <em>could</em> modify the function to run values from 0.5 to -0.5 or similar to take the peaks off and reduce the discomfort, however, where's the fun in that? Let's route the noise we've created through a filter.</p>
+<p>여러분은 이것이 꽤 쉭쉭하는 소리가 나거나 금속성의 소리가 나는 것을 인지하였을 것입니다. 우리는 백색소음을 만들었고, 백색소음이라면 그래야만 합니다. 값은 -1과 1사이에서 실행되는데, 이는 우리가 모든 주파수의 피크를 가지고 있는데, 결국 이는 실제로 꽤 찢어지는 듯한 소리임을 의미합니다. 우리는 이 함수가 0.5에서 -0.5에서 실행되거나 피크를 내려 불편함을 줄이는 유사한 작업을 하도록 수정<em>할 수 있는데</em>, 그렇게 하면 무엇이 즐겁나요? 우리가 생성한 노이즈가 필터를 통과하도록 전송합시다.</p>
```

Comment:
```suggestion
<p>여러분은 이것이 꽤 쉭쉭하는 소리가 나거나 금속성의 소리가 나는 것을 인지하였을 것입니다. 우리는 백색소음을 만들었고, 백색소음이라면 그래야만 합니다. 값은 -1과 1사이에서 실행되는데, 이는 우리가 모든 주파수의 피크를 가지고 있는데, 결국 이는 실제로 꽤 찢어지는 듯한 소리임을 의미합니다. 우리는 이 함수가 0.5에서 -0.5에서 실행되거나 피크를 내려 불편함을 줄이는 유사한 작업을 하도록 수정<em>할 수 있는데</em>, 그렇게 하면 재미가 없죠? 우리가 생성한 노이즈가 필터를 통과하도록 전송합시다.</p>
```
where's the fun in that을 조금 더 이해하기 쉽도록 의역해보았습니다 !